packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -57,7 +57,12 @@ function ListTable(props) {
 			id={id}
 			onRowClick={onRowClickCallback}
 			noRowsRenderer={() => <NoRows t={t} />}
-			rowClassName={classNames(rowThemes)}
+			rowClassName={({ index }) => {
+				if (collection[index]) {
```

Comment:
Why are you testing the truthiness of `collection[index]` ?
Isn't `classNames(rowThemes, collection[index].className)` sufficient ?